### DISCUSSION TITLE
I ran trivy image and got an error

### DISCUSSION THREAD

**Discussion by wangfan0524** [aquasecurity/trivy#7994] [Q&A]
### Question

Hello,I ran trivy image and got an error,The image is built locally through docker build. what should i do? thanks
<img width="1436" alt="image" src="https://github.com/user-attachments/assets/79fb5000-0bfe-4151-b1bc-79940c3afdca">

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Operating System

macOS sonoma

### Version

```bash
0.57.1
```

**Comment by nikpivkin** [comment]
Hi @wangfan0524 !

You must enable `podman.sock` before scanning https://trivy.dev/v0.57/docs/target/container_image/#podman
